src/main/java/frc/robot/RobotContainer.java

```diff
@@ -76,9 +79,10 @@ public RobotContainer()
   private void configureBindings() {
    
     m_driverController.x().onTrue((new InstantCommand(drivebase::zeroGyro)));
```

Comment:
You might want to re-order these statements so that it's `a`, `b`, `x`, `y` - it'll make the mapping a bit more clear if they're grouped together and ordered

Reply:
Thanks!